arkouda/dataframe.py

```diff
@@ -217,25 +223,26 @@ def __init__(self, initialdata=None):
             # Update the dataframe indices and metadata.
             if len(sizes) > 0:
                 self._size = sizes.pop()
-            # If the index column was passed in, use that instead of
+
+            # If the index param was passed in, use that instead of
             # creating a new one.
-            if self.data['index'] is None:
-                self.data['index'] = arange(0, self._size, 1)
+            if self.index is None:
+                self._set_index(arange(0, self._size, 1))
```

Comment:
i know this was existing code and certainly does not need to be changed but I wanted to mention it. the start and stride don't need to be specified here since it's the default `arange(self._size)` should work. This happens in a couple of places

Reply:
I left this for now since it is not impacting functionality. I added issue #1236 to track.